DEVELOPMENT.md

```diff
@@ -6,7 +6,7 @@ This guide outlines the steps to set up a development environment for Dojo. If y
 
 - [Rust](https://github.com/rust-lang/rust)
 - [Cairo](https://github.com/starkware-libs/cairo)
-- [protoc](https://github.com/protocolbuffers/protobuf)
+- [Protobuf](https://github.com/protocolbuffers/protobuf)
```

Comment:
i think `protoc` is more accurate here though as we need user to have the `protoc` binary but i understand that it's not consistent with the other two. so we're not gonna accept this correction.